src/components/Breadcrumbs/Breadcrumb/Breadcrumb.scss

```diff
@@ -7,16 +7,6 @@
   white-space: nowrap;
   overflow: hidden;
 
-  > a {
```

Comment:
I have removed this since the styling should be handled by the children component (<ControlLink /> as standard)